docs/reference/migration/migrate_8_0/indices.asciidoc

```diff
@@ -6,6 +6,24 @@
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[[deprecation-system-indices]]
+.Direct access to system indices is deprecated.
+[%collapsible]
+====
+*Details* +
+Directly accessing system indices is deprecated, and may be prevented in a
+future version. If you must access a system index, create a security role with
+an index permission that targets the specific index and set the
+`allow_restricted_indices` permission to `true`. Refer to 
+{ref}/defining-roles.html#roles-indices-priv[indices privileges] for
+information on adding this permission to an index privilege.
+
+*Impact* +
+Accessing system indices directly results in warnings in the header of API
+responses. Use {kib} or the associated feature's {es} APIs to manage the data
```

Comment:
```suggestion
responses. If available, use {kib} or the associated feature's {es} APIs to manage the data
```

We may not have coverage for every feature, so we should mention to check availability.